.fixtures.yml

```diff
@@ -7,7 +7,7 @@ fixtures:
       repo: 'https://github.com/ghoneycutt/puppet-module-rpcbind.git'
       ref: 'v3.0.0'
     nfs:
-      repo: 'https://github.com/ghoneycutt/puppet-module-nfs.git'
-      ref: 'v2.3.0'
+      repo: 'https://github.com/kodguru/puppet-module-nfs.git'
+      ref: 'v2.4.0'
```

Comment:
Lately I did start to remove the `ref`erence so that the latest version is always used. This will show problems with newer versions of dependency modules quicker. It's also quick to add/change this while developing.
The versions that are mentioned in `metadata.json` are the versions that should be used in production.

Reply:
Do you propose we remove refs? Maybe it should be its own PR though.

Reply:
It is only used for developing, so why not remove it and always use the newest code.